modules/storages/app/controllers/storages/admin/storages/project_storages_controller.rb

```diff
@@ -81,7 +81,21 @@ def destroy_confirmation_dialog
   end
 
   def destroy
-    @project_storage.destroy
+    result = Storages::ProjectStorages::DeleteService
+      .new(user: current_user, model: @project_storage)
+      .call
+
+    # rubocop:disable Rails/ActionControllerFlashBeforeRender
+    result.on_success do
+      flash[:primer_banner] = { message: I18n.t(:notice_successful_delete) }
+    end
+
+    result.on_failure do |failure|
+      error = failure.errors.map(&:message).join("; ")
+      flash[:primer_banner] = { message: t("project_storages.remove_project.deletion_failure_flash", error:), scheme: :danger }
```

Comment:
🟠 Q: Does i18n/Active model errors have special handling for `error:`? AFAICT, the `deletion_failure_flash` does not interpolate the errors and the `;` join seems odd to me.

Reply:
Good catch! Actually I tried testing this one but wasn't able to create an error like this artificially.